src/containers/DeleteImageContainer.js

```diff
@@ -0,0 +1,20 @@
+import { connect } from 'react-redux';
+import { DeleteImageButton } from '../views/DeleteImageView';
+import { deleteImage } from '../actions/actions';
+
+function mapStateToProps(state) {
+  return {
+    currentImage: state.currentImage,
+  };
+}
+
+function mapDispatchToProps(dispatch) {
+  return {
+    deleteImage: id => dispatch(deleteImage(id)),
+  };
+}
```

Comment:
can use object notation here as well, if the wrapped function looks like:

```js
(foo, bar, baz) => dispatch(someAction(foo, bar, baz))
```

the object format will all parameters to the action